### PR TITLE
Update Stormworks

### PR DESCRIPTION
**Pull request by delphyinium**
# Description
This is a fix that was tested in the issue mentioned #1851 

Changes:

- change image from wine_latest to wine_staging
- update readme to address console output and tps misreporting
- updated install script to add echo message and removed unneeded package install step
closes https://github.com/parkervcp/eggs/issues/1851

## Checklist for all submissions

* [X] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [X] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### CODE REVIEW

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]
Use the panel export feature. You dont edit the json file directly. Then the egg will update to pltv_2 format if your panel is up to date

**Comment by delphyinium** [comment]
> Use the panel export feature. You dont edit the json file directly. Then the egg will update to pltv_2 format if your panel is up to date

Apologies. Fixed. I went through the effort of deleting the file and uploading the new one when I could have just edited the existing file 🤦 
Should be good now though.

**Comment by QuintenQVD0** [comment]
This egg needs a extra variable to auto update steamcmd. And in the install script can be upate as the apt update and the apt install is not needed because it uses the install image that has all those packages

**Comment by delphyinium** [comment]
> This egg needs a extra variable to auto update steamcmd. And in the install script can be upate as the apt update and the apt install is not needed because it uses the install image that has all those packages. And the new echo done at the end of install script. If you don't want to do that I will do that for you tommorow.

I believe I have fixed everything. I tested Auto-Update to verify it checks for an update, which it does. I also got rid of the apt install, since it is not required like you said, and added an echo at the end to signal it has completed installation.

File: game_eggs/steamcmd_servers/stormworks/egg-stormworks--build-and-rescue.json [hunk 1]
```diff
@@ -49,6 +49,16 @@
...
             "field_type": "text"
+        },
+        {
+            "name": "AUTO_UPDATE",
+            "description": "This will auto-update SteamCMD on boot.\r\n0 is disabled, 1 is enabled.",
+            "env_variable": "AUTO_UPDATE",
+            "default_value": "0",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|string|max:20",
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/stormworks/egg-stormworks--build-and-rescue.json:60]: This should be `required|boolean`

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

File: game_eggs/steamcmd_servers/stormworks/egg-stormworks--build-and-rescue.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    "exported_at": "2022-10-26T12:10:19-06:00",
+    "name": "Stormworks: Build and Rescue",
+    "author": "iamkubi@gmail.com",
+    "description": "Join a world where you design, create and pilot your own air sea rescue service. Release your inner hero as you battle fierce storms out at sea to rescue those in need.",
+    "features": [
+        "steam_disk_space"
+    ],
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:wine_staging": "ghcr.io\/parkervcp\/yolks:wine_staging"
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/stormworks/egg-stormworks--build-and-rescue.json:15]: Please use `Wine_staging|<url>` in the panel docker image so then it when it export it only shows wine staging to the end user.

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

**Comment by QuintenQVD0** [comment]
If the eggs then can be tested by someone or I will do it in the weekend then I will aprove this.

**Comment by delphyinium** [comment]
> If the eggs then can be tested by someone or I will do it in the weekend then I will aprove this. 

Ok, I can fix these in a bit, then I'll test and get back to you.

**Comment by delphyinium** [comment]
Sorry for the delay, I quickly made the changes you requested. I tested and it seems to be running smooth. Once again apologies for the delay.

**Review by QuintenQVD0 — APPROVED** [review]

**Comment by QuintenQVD0** [comment]
Np, LGTM!

**Pull request merged**